src/hooks/useCroct.ssr.test.tsx

```diff
@@ -1,12 +1,17 @@
 import {renderHook} from '@testing-library/react-hooks/server';
 import {ReactNode} from 'react';
+import {Plug} from '@croct/plug';
 import {useCroct} from './useCroct';
 import {CroctProvider} from '../CroctProvider';
 
+type RenderHookProps = {
```

Comment:
Is it really needed? What is the error without this?

Reply:
`Type '({ children }: { children: any; }) => Element' is not assignable to type 'WrapperComponent<unknown> | undefined'.`